sdcore-helm-charts/Chart.yaml

```diff
@@ -19,31 +19,26 @@ maintainers:
 dependencies:
   - name: omec-control-plane
     version: 0.13.0
-    repository: https://charts.aetherproject.org
-    #repository: "file://../omec-control-plane"
+    repository: "file://../omec-control-plane"
     condition: omec-control-plane.enable4G
 
   - name: omec-sub-provision
     version: 1.0.0
-    repository: https://charts.aetherproject.org
-    #repository: "file://../omec-sub-provision"
+    repository: "file://../omec-sub-provision"
     condition: omec-sub-provision.enable
 
   - name: 5g-control-plane
     version: 1.0.1
-    repository: https://charts.aetherproject.org
-    #repository: "file://../5g-control-plane"
+    repository: "file://../5g-control-plane"
     condition: 5g-control-plane.enable5G
 
   - name: bess-upf
     version: 1.0.0
-    repository: https://charts.aetherproject.org
-    #repository: "file://../bess-upf"
+    repository: "file://../bess-upf"
     alias: omec-user-plane
     condition: omec-user-plane.enable
 
   - name: 5g-ran-sim
     version: 1.0.0
```

Comment:
Is version required in this case?
Seconf question can we not keep commented repository as is.

Reply:
1) Yes, it is required.
2) Do you mean something like this? I would not like to have it as it was before because when using local charts, it will always check for the dependency Charts (5g-control-plane, bess-upf, etc.) from their online repository and not the local charts themselves
```
    #repository: https://charts.aetherproject.org
    repository: "file://../bess-upf"
```